cmd/operator/generate.sh

```diff
@@ -16,8 +16,8 @@ cd ${SCRIPT_DIR}
 echo "Generating CRD."
 ${LOCALBIN}/controller-gen crd paths="./..." output:dir=../../charts/kubearchive/crds
 
-patch="/tmp/crd.$$"
-cat << EOF > $patch
+PATCH=$(mktemp -t crd.XXXXXXXX)
+cat << EOF > $PATCH
```

Comment:
Several [styleguides](https://google.github.io/styleguide/shellguide.html#s5.6-variable-expansion) suggest the usage of the [shell expansion format](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) in variables over the plain usage of variable (meaning ${PATCH} better than $PATCH.

Given that we are already using the expansion format for some variables (like `CRD`) I suggest we keep consistency with it and use it in the rest:

```suggestion
cat << EOF > ${PATCH}
```

Reply:
Done!